### DISCUSSION TITLE
Welcome to YOLO-World Discussions!

### DISCUSSION THREAD

**Discussion by wondervictor** [AILab-CVC/YOLO-World#117] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fgd0707** [comment]
网址为http://0.0.0.0:8080/ 的页面可能存在问题，或者已永久移动到新的网址。demo.py运行会这样，可以帮我解答一下吗大哥

**Reply by laiaqwq** [reply]
试一下 http://127.0.0.1:8080/
